deploy/crds/deprecated/core_v1_storagecluster_crd.yaml

```diff
@@ -878,6 +878,14 @@ spec:
                     remoteWriteEndpoint:
                       type: string
                       description: Specifies the remote write endpoint for Prometheus.
+                    alertManager:
```

Comment:
when do we need to update this deprecated crd.yaml? for the container resources one I didn't

Reply:
It will only be used for k8s cluster < 1.16. 
It is not necessary to add to the deprecated schema file. I just added because I remembered.